test/tornado_tests/test_motor_core.py

```diff
@@ -80,7 +80,7 @@ def attrs(klass):
 
 pymongo_cursor_only = set([
     'count',
-    'retrieved']).union(pymongo_only)
```

Comment:
I don't think this is quite right. We should keep the union here and adjust `pymongo_only` to be correct (assuming we want to keep the newly added `next()`).

Reply:
We can't really do that since `pymongo_only` is also used to compare attrs of `collection` and `database` objects - both of which have a `next` method in PyMongo but not in Motor.

Reply:
On a closer look, it is unclear to me why those classes have `next` in PyMongo - they simply raise `TypeError` - but that isn't something that can be litigated on this PR.

Reply:
Ack.